### PR TITLE
Changed config registry using

### PR DESCRIPTION
**Pull request by segabriel**

### CODE REVIEW

File: scalecube-configuration/src/main/java/io/scalecube/configuration/repository/couchbase/CouchbaseSettings.java [hunk 1]
```diff
@@ -26,42 +19,40 @@
   private static final String BUCKET_INDEX_REPLICAS =
       "bucket.indexReplicas";
   private static final String BUCKET_ENABLE_FLUSH = "bucket.enableFlush";
-  private final Properties settings;
   private final CouchbaseProperties couchbaseProperties;
   private List<String> clusterNodes;
   private List<String> roles;
 
+  private ConfigRegistry configRegistry = ConfigRegistryConfiguration.configRegistry();
```
Comment (dmytro-lazebnyi) [inline scalecube-configuration/src/main/java/io/scalecube/configuration/repository/couchbase/CouchbaseSettings.java]: final?

File: scalecube-configuration/src/main/java/io/scalecube/configuration/repository/couchbase/CouchbaseSettings.java [hunk 2]
```diff
@@ -26,42 +19,40 @@
...
       "bucket.indexReplicas";
   private static final String BUCKET_ENABLE_FLUSH = "bucket.enableFlush";
-  private final Properties settings;
   private final CouchbaseProperties couchbaseProperties;
   private List<String> clusterNodes;
   private List<String> roles;
 
+  private ConfigRegistry configRegistry = ConfigRegistryConfiguration.configRegistry();
+
+  public static void main(String[] args) {
```
Comment (dmytro-lazebnyi) [inline scalecube-configuration/src/main/java/io/scalecube/configuration/repository/couchbase/CouchbaseSettings.java]: redundant main

File: scalecube-configuration/src/main/java/io/scalecube/configuration/repository/couchbase/CouchbaseSettings.java [hunk 3]
```diff
@@ -79,25 +70,11 @@ String couchbaseAdminPassword() {
...
-    return getProperty(NEW_BUCKET_NAME_FORMAT);
-  }
-
-
-  private String getProperty(String key) {
-    return settings.getProperty(key);
+    return configRegistry.stringValue(NEW_BUCKET_NAME_FORMAT, null);
   }
 
   static class Builder {
```
Comment (dmytro-lazebnyi) [inline scalecube-configuration/src/main/java/io/scalecube/configuration/repository/couchbase/CouchbaseSettings.java]: why do we need this builder?

File: scalecube-configuration/src/main/java/io/scalecube/configuration/repository/couchbase/CouchbaseSettings.java [hunk 4]
```diff
@@ -79,25 +70,11 @@ String couchbaseAdminPassword() {
...
-    return roles;
-  }
-
-  private List<String> getList(String key, List<String> list) {
-    if (list == null) {
-      String value = getProperty(key);
-      list = value.length() > 0 ? Arrays.asList(value.split(",")) : new ArrayList<>();
-    }
-    return list;
+    return configRegistry.stringListValue(BUCKET_ROLES, Collections.emptyList());
```
Comment (dmytro-lazebnyi) [inline scalecube-configuration/src/main/java/io/scalecube/configuration/repository/couchbase/CouchbaseSettings.java]: Collections.unmodifiableList

**Review by artem-v — APPROVED** [review]

**Pull request merged**